schemas/session-replay/common/_slot-supported-common-record-schema.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "session-replay/common/_slot-supported-common-record-schema.json",
+  "title": "SlotSupportedCommonRecordSchema",
+  "type": "object",
+  "description": "Schema of common properties for a Record event type that is supported by slots.",
+  "allOf": [
+    {
+      "$ref": "_common-record-schema.json"
+    },
+    {
+      "properties": {
+        "slotId": {
+          "type": "string",
```

Comment:
Discussed with @mariusc83 string are more reliable that integer (cf. int64 non supported long number in javascript issue)